<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLRuleWidget</class>
 <widget class="QWidget" name="PCLRuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelProgram">
       <property name="text">
        <string>Programme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelParam">
       <property name="text">
        <string>Paramètre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSubParam">
       <property name="text">
        <string>Sous Paramètre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIO">
       <property name="text">
        <string>I/O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelClasse">
       <property name="text">
        <string>Classe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
